daemon/models/custom.py

```diff
@@ -59,6 +59,8 @@ class _PydanticConfig(BaseConfig):
 
 
 def build_pydantic_model(model_name: str, module: str):
+    from jina.parsers import helper
+    helper._SHOW_ALL_ARGS = True
```

Comment:
isn't this change too global? I think it should be handled via an `argument` in the parsers or removed.